### PR TITLE
EN-2378 DE filter improvements

### PR DESCRIPTION
**Pull request by zonia3000**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/digitalexchange/component/DigitalExchangeComponentListProcessor.java [hunk 1]
```diff
@@ -71,18 +73,18 @@ public DigitalExchangeComponentListProcessor(RestListRequest restListRequest, St
         return sort -> {
             switch (sort) {
                 case LAST_UPDATE:
-                    return (a, b) -> a.getLastUpdate().compareTo(b.getLastUpdate());
+                    return (a, b) -> ObjectUtils.compare(a.getLastUpdate(), b.getLastUpdate());
                 case VERSION:
-                    return (a, b) -> a.getVersion().compareToIgnoreCase(b.getVersion());
+                    return (a, b) -> StringUtils.compare(a.getVersion(), b.getVersion());
```
Comment (Kerruba) [inline plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/digitalexchange/component/DigitalExchangeComponentListProcessor.java]: Why the version requires exact match and not ignore case?

Comment (zonia3000) [reply plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/digitalexchange/component/DigitalExchangeComponentListProcessor.java]: Thanks, it's a mistake.

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by zonia3000** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by zonia3000** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by zonia3000** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by zonia3000** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by zonia3000** [comment]
retest this please

**Comment by entando-service** [comment]
All is Well

**Pull request merged**